packages/fractor/src/Testing/PHPUnit/StaticPHPUnitEnvironment.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\Fractor\Testing\PHPUnit;
+
+final class StaticPHPUnitEnvironment
+{
+    public static function isPHPUnitRun(): bool
+    {
+        return \defined('PHPUNIT_COMPOSER_INSTALL');
```

Comment:
Does this also work if PHPUnit is installed as a PHAR?

Reply:
hmm, didn't test. do you have a testing environment with a phar?

Reply:
Looks like that would not work, at least according to what I found here: https://github.com/sebastianbergmann/phpunit/blob/b856d6a9b2be1387899d0366f783cd1aad0bd7df/src/Framework/TestRunner/templates/method.tpl#L29 Not sure if that is also true for the general execution.

Let's leave it like this for now and if we see problems with PhpUnit, we can still fix this.